### DISCUSSION TITLE
iOS Shader에서 Texture2DArray가 이상동작 하는것같습니다.

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#453] [5. Q&A]
Shader 내에서 텍스쳐 샘플링을 8개 이상 해야할것같아 
Texture2DArray를 사용하였더니 랜더링하는데 문제가 생깁니다.

기존에 iOS에도 있던 Auto Graphics API 옵션이 사라졌는데
이제 기본적으로 Metal로 빌드가 되는건지 궁금합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 관련 코드 및 구조를 올려주시면 문제 해결에 도움을 드릴 수 있을 것 같습니다. 
감사합니다.
